### DISCUSSION TITLE
Wii classic controller support

### DISCUSSION THREAD

**Discussion by Adx431** [darthcloud/BlueRetro#142] [Q&A] [answered]
Is the Wii classic controller supported? 
I have successfully paired the Wii mote, but it doesn't recognize any inputs from the classic controller.
Sorry if this has been answered somewhere else, I just haven't been able to get it working.
Thanks

**Comment by darthcloud** [comment, marked as answer]
Yes it is supported, which one are you using normal or pro?

**Reply by Adx431** [reply]
I'm using the pro

**Reply by darthcloud** [reply]
It got broken in v1.2.1, can you test in version v1.2?

 I only got normal so just want to confirm pro work too in v1.2.

**Reply by Adx431** [reply]
Yes, I just tested it with version 1.2 and it works perfectly with my classic controller pro.
Thank you very much, and I hope this issue can be solved in the future.

**Reply by darthcloud** [reply]
I'll probably spin a v1.2.2 within a week. Thanks for testing!!

**Reply by Adx431** [reply]
That would be awesome, thanks :B

**Reply by darthcloud** [reply]
Fixed in v1.2.2 -> https://github.com/darthcloud/BlueRetro/releases/tag/v1.2.2

**Reply by Adx431** [reply]
That's awesome thanks, already tested it 😎

**Comment by CheaMJ** [comment]
Hola, disculpa podrías subir alguna foto de como van los cables al control?

Hello, sorry, could you upload a photo of how the cables go to the control?
